### PR TITLE
Add proper extension to all Perl scripts

### PR DESCRIPTION
**Pull request by cmadjar**
**Added `.pl` extension to the following scripts:**
- `batch_uploads_imageuploader`
- `batch_uploads_tarchive`
- `tarchiveLoader` 
- `uploadNeuroDB/bin/Mincinfo_wrapper` 
- `uploadNeuroDB/bin/mincpik` 
- `profileTemplate` 
- `tools/seriesuid2fileid`

**Caveat for existing projects**
Projects that call in a cronjob one of the scripts mentioned above (most likely `batch_uploads_imageuploader` and `batch_uploads_tarchive`) will need to update their cronjob to include the `.pl` extension in their command. 

**Notes:**
- `find_uploads_tarchive` file was not updated with a `.pl` extension since it is to be removed by another PR.
- `Mincinfo_wrapper` and `mincpik` present in `uploadNeuroDB/bin` have symlink present in the main directory of `bin/mri`. The symlink have been updated to point to the new file names with the `.pl` extension. Will need to investigate in another PR whether we could get rid of the symlinks and add the path `uploadNeuroDB/bin` in $PATH (in the environment file)

### CODE REVIEW

**Comment by cmadjar** [comment]
@MounaSafiHarab I assigned this to you since you started working on this for tarchiveLoader during the documentation blitz.
@nicolasbrossard if you have time testing it, that would be awesome!!

**Comment by cmadjar** [comment]
@nicolasbrossard Added the missing `.pl` extension to `tarchiveLoader` call in `tools/example_scripts/deletemincsqlwrapper.pl`. Ready for another round of review when you have time. Thanks!!

File: uploadNeuroDB/tarchiveLoader.pl [hunk 1]
```diff
@@ -216,7 +216,7 @@
 my $User             = `whoami`;
 
 # fixme there are better ways 
-my @progs = ("convert", "Mincinfo_wrapper", "mincpik", $converter);
+my @progs = ("convert", "Mincinfo_wrapper.pl", "mincpik", $converter);
```
Comment (nicolasbrossard) [inline uploadNeuroDB/tarchiveLoader.pl]: Shouldn't it be mincpik.pl here?

File: tools/example_scripts/deletemincsqlwrapper.pl [hunk 2]
```diff
@@ -113,7 +113,7 @@
 
     if ($insertminc) {
       # Running tarchiveLoader on the archived tar as a whole will only insert new minc files that are not already in the files table
```
[the remark this answers is not in the document]
Comment (cmadjar) [reply tools/example_scripts/deletemincsqlwrapper.pl]: Good catch!! Resolved now

**Comment by cmadjar** [comment]
@nicolasbrossard Thank you for the feedback! I have integrated them.

**Comment by cmadjar** [comment]
@nicolasbrossard @MounaSafiHarab I rebased the branch to minor as it had conflicts. Feel free to review it again if you wish (the rebase was pretty straightforward)

**Pull request merged**